### PR TITLE
adds a couple alternative moloch strategies

### PR DESCRIPTION
**Pull request by scottrepreneur**
Changes proposed in this pull request:
- moloch strategies for
  - include all loot and share holders
  - only loot holders

### CODE REVIEW

File: src/strategies/moloch-all/index.ts [hunk 1]
```diff
@@ -0,0 +1,126 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'adridadou';
```
Comment (bonustrack) [inline src/strategies/moloch-all/index.ts:4]: This should be your GitHub username
```suggestion
export const author = 'scottrepreneur';
```

File: src/strategies/moloch-loot/index.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'adridadou';
```
Comment (bonustrack) [inline src/strategies/moloch-loot/index.ts:4]:
```suggestion
export const author = 'scottrepreneur';
```

**Comment by bonustrack** [comment]
Hi! Thanks for your PR, can you add the strategies in this file too https://github.com/snapshot-labs/snapshot.js/blob/master/src/strategies/index.ts ? Once you do that you can test the strategy using `npm run test --strategy=moloch-all`

**Comment by scottrepreneur** [comment]
these tests are working now @bonustrack if you can give it another shot. thanks! 🙏

**Review by bonustrack — APPROVED** [review]
Looks good!

**Comment by amsos** [comment]
Yoo, this is awesome.  Thanks for adding these strategies!

**Pull request merged**